lib/state/settings/reducer.js

```diff
@@ -35,6 +37,10 @@ function reducer(state = initialState, action) {
       return { ...state, markdownEnabled: action.markdownEnabled };
     case 'setNoteDisplay':
       return { ...state, noteDisplay: action.noteDisplay };
+    case 'setSearchSortReversed':
+      return { ...state, searchSortReversed: action.searchSortReversed };
+    case 'setSearchSortType':
+      return { ...state, searchSortType: action.searchSortType };
```

Comment:
we would have good value in using a new reducer in a search directory where we can store all the filtering and sort properties. these reducers used to be separate but they were merged and that can cause issues, especially with regards to handling more than one action type in a given reducer. it might be worth our time to decouple these as they once were but that would be a job for another PR

you can see what I was exploring in https://github.com/Automattic/simplenote-electron/pull/1725/files#diff-a36f9dcd572ee4de0417fd7a1f181542